MvvmCross.Plugins/Color/Platforms/Android/Plugin.cs

```diff
@@ -4,17 +4,17 @@
 
 using MvvmCross.Binding.Bindings.Target.Construction;
 using MvvmCross.Plugin.Color.Platforms.Android.BindingTargets;
-using MvvmCross.Plugin;
 using MvvmCross.UI;
 
 namespace MvvmCross.Plugin.Color.Platforms.Android
 {
     [MvxPlugin]
     [Preserve(AllMembers = true)]
-    public sealed class Plugin : IMvxPlugin
+    public sealed class Plugin : BasePlugin, IMvxPlugin
```

Comment:
No need to explicitly stated that it inherits from IMvxPlugin, though this is not really a problem

Reply:
👍 right!